File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { Ec2ShopSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await Ec2ShopSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- Ec2Shop SDK exists test

local sdk = require("ec2-shop_sdk")

describe("Ec2ShopSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
